### DISCUSSION TITLE
Fetch only modified/updated items for tabular view for better experience - instead of all items

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#13464] [Feature Requests]
Our tabular view fetches all the items through API based on the filter applied if we keep 1000 items per page,

most of the time. it takes time to load even there is no change in all the items.

why not enable API which fetch only updated items instead of fetching all items?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
